### PR TITLE
+ [jsfm] add frameworks config for custom framework build

### PR DESCRIPTION
**Pull request by yuanyan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,6 +76,7 @@
     "scroll-to": "0.0.2",
     "semver": "^5.1.0",
     "weex-vue-framework": "2.0.5-weex.1",
+    "weex-rx-framework": "0.1.13",
```
Comment (Jinjiang) [inline package.json]: It's better to separate the commit with two parts. 1 is adding frameworks config script. 2 is adding rx dep. And the second one is recommended to go to `jsfm-feature-0.18` first. We can have a face-to-face talk later. Don't mind about this :-)

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
   },
   "scripts": {
     "postinstall": "bash ./bin/install-hooks.sh",
+    "build:frameworks:config": "node build/frameworks.config.js",
```
Comment (yuanyan) [inline package.json]: @Jinjiang `build:config` maybe another script for all config about, naming `frameworks:config` is more accurate now.

File: package.json (hunk 1 above)
Comment (yuanyan) [inline package.json]: I dont think so, `build:frameworks:config` will scan `package.json` file, the dep of weex-rx-framework is part of the commit.

**Comment by Hanks10100** [comment]
After a discussion, this feature is deprecated.

**Pull request closed without merging**